src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,30 @@
+package duke.task;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
```

Comment:
Avoid leaving empty lines, which is inconsistent with the layout. In this case 1 line break is enough.